app/assets/stylesheets/components/common/_contact_panel.scss

```diff
@@ -1,27 +1,37 @@
-// Contact Panel
-//
-// Styleguide Contact Panel
-
```

Comment:
🙌 This multi-line comment needs to be added back, since it instructs the [KSS parser to read sections](https://github.com/kneath/kss/blob/b0791708cb397f5e8995a855af0d1a8b0aa89a2e/lib/kss/comment_parser.rb#L119-L120) from this stylesheet in order to be displayed properly in the guideline available at `/en/styleguide/components/website`.

After that's done, tests will go green.